third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -3737,7 +3740,7 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
                     loc,
                     &format!(
                         "`{}` is a function and not a macro",
-                        qsym.display(self.env())
+                        qsym.display_simple(self.env())
```

Comment:
Check the error message and it makes more sense to print the function name only.